vscode/src/webview/rePage.tsx

```diff
@@ -96,32 +60,31 @@ export function RePage(props: {
         ) : null}
         <h1>Azure Quantum Resource Estimator</h1>
       </div>
-      <details open>
-        <summary style="font-size: 1.5em; font-weight: bold; margin: 24px 8px;">
-          Results
-        </summary>
-        <ResultsTable
-          columnNames={columnNames}
-          data={props.estimatesData}
-          initialColumns={initialColumns}
-          ensureSelected={true}
-          onRowSelected={onRowSelected}
-          onRowDeleted={onRowDeleted}
-        />
-      </details>
+      <Summary
```

Comment:
We might want a different name than `Summary` for the custom component, as `summary` is a standard HTML tag that is also used in this code (and is something very different). I was confused for a second until I realized that distinction.